modules/features/endofyear/src/main/java/au/com/shiftyjelly/pocketcasts/endofyear/StoriesViewModel.kt

```diff
@@ -50,11 +50,13 @@ class StoriesViewModel @Inject constructor(
     init {
         viewModelScope.launch {
             storiesDataSource.loadStories().stateIn(viewModelScope).collect { result ->
-                stories.clear()
+                cancelTimer()
```

Comment:
I was able to reproduce the story speed issue by opening stories page during account sync.

Since `loadStories()` returns a flow of stories by combining query results from the DB and queries may return different results during account sync (e.g. `listeningTime` may change every time an episode is added to the DB), the result might be collected several times. I cancel the timer here so that only one timer is started at a time.